runtime/doc/cmdline.txt

```diff
@@ -1096,12 +1096,15 @@ for an Ex command, then two mappings will be added to use <Tab> for completion
 in the command-line window, like this: >
 	:imap <buffer> <Tab> <C-X><C-V>
 	:nmap <buffer> <Tab> a<C-X><C-V>
-Note that hitting <Tab> in Normal mode will do completion on the next
+As of this moment, this mapping is hardcoded and comes from `ex_getln.c`.
```

Comment:
this is too much detail . It is a mapping like any other, its origin is irrelevant.